test/org/joml/test/IntersectiondTest.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2016-2020 JOML.
```

Comment:
Please use: `Copyright (c) 2020 JOML.`
All copyrights in a file start with the introduction of that particular file.